packages/postgres/package.json

```diff
@@ -27,11 +26,8 @@
   },
   "main": "./dist/index.cjs",
   "module": "./dist/index.js",
```

Comment:
There seems to be another issue when trying to actually release this change, as I tested it out with a snapshot release

https://github.com/vercel/storage/actions/runs/6222845972/job/16887566314#step:8:46

```
Error: 🦋  error src/index.ts(4,28): error TS7016: Could not find a declaration file for module '@vercel/postgres'. '/home/runner/work/storage/storage/packages/postgres/dist/index.cjs' implicitly has an 'any' type.
🦋  error   Try `npm i --save-dev @types/vercel__postgres` if it exists or add a new declaration (.d.ts) file containing `declare module '@vercel/postgres';`
Error: 🦋  error src/index.ts(5,[47](https://github.com/vercel/storage/actions/runs/6222845972/job/16887566314#step:8:48)): error TS7016: Could not find a declaration file for module '@vercel/postgres'. '/home/runner/work/storage/storage/packages/postgres/dist/index.cjs' implicitly has an 'any' type.
🦋  error   Try `npm i --save-dev @types/vercel__postgres` if it exists or add a new declaration (.d.ts) file containing `declare module '@vercel/postgres';`
🦋  error 
🦋  error Error: error occured in dts build
🦋  error     at Worker.<anonymous> (/home/runner/work/storage/storage/node_modules/.pnpm/tsup@7.2.0_typescript@5.2.2/node_modules/tsup/dist/index.js:2294:26)
🦋  error     at Worker.emit (node:events:514:28)
🦋  error     at MessagePort.<anonymous> (node:internal/worker:243:53)
🦋  error     at [nodejs.internal.kHybridDispatch] (node:internal/event_target:741:20)
🦋  error     at exports.emitMessage (node:internal/per_context/messageport:23:28)
🦋  error DTS Build error
🦋  error RollupError: Failed to compile. Check the logs above.
🦋  error     at error (/home/runner/work/storage/storage/node_modules/.pnpm/rollup@3.21.2/node_modules/rollup/dist/shared/rollup.js:279:30)
🦋  error     at Object.error (/home/runner/work/storage/storage/node_modules/.pnpm/rollup@3.21.2/node_modules/rollup/dist/shared/rollup.js:2[48](https://github.com/vercel/storage/actions/runs/6222845972/job/16887566314#step:8:49)30:20)
🦋  error     at Object.error (/home/runner/work/storage/storage/node_modules/.pnpm/rollup@3.21.2/node_modules/rollup/dist/shared/rollup.js:24024:42)
🦋  error     at generateDtsFromTs (/home/runner/work/storage/storage/node_modules/.pnpm/tsup@7.2.0_typescript@5.2.2/node_modules/tsup/dist/rollup.js:7512:22)
🦋  error     at /home/runner/work/storage/storage/node_modules/.pnpm/tsup@7.2.0_typescript@5.2.2/node_modules/tsup/dist/rollup.js:7519:59
🦋  error     at _nullishCoalesce (/home/runner/work/storage/storage/node_modules/.pnpm/tsup@7.2.0_typescript@5.2.2/node_modules/tsup/dist/rollup.js:1:198)
🦋  error     at Object.transform (/home/runner/work/storage/storage/node_modules/.pnpm/tsup@7.2.0_typescript@5.2.2/node_modules/tsup/dist/rollup.js:7519:18)
🦋  error     at /home/runner/work/storage/storage/node_modules/.pnpm/rollup@3.21.2/node_modules/rollup/dist/shared/rollup.js:2[50](https://github.com/vercel/storage/actions/runs/6222845972/job/16887566314#step:8:51)29:40
```

Reply:
Looks like the `prepublish` script does not take into account the workspace dependency between `@vercel/postgres` and `@vercel/postgres-kysely`.

Reply:
Oh right, like we should first build vercel/postgres and then postgres-kysely, correct?

Reply:
Yeah, the normal release process uses `turbo run build` which takes care of that.

@dferber90 are you working on this?

Reply:
🤦 Yeah the `pnpm build` was totally missing. Added in https://github.com/vercel/storage/pull/393 and it is working now. Thanks for pointing that out!